front/src/components/app/index.js

```diff
@@ -22,6 +23,7 @@ class App extends Component {
                     <Route path="/forgotPassword/" component={ForgotPasswordPage} />
                     <Route path="/groups" component={GroupsPage} />
                     <Route path="/tasks" component={TasksList} />
+                    <Route path="/changePassword" component={ChangePasswordButtons} />
```

Comment:
Why did you call it `ChangePasswordButtons`? I don't think that we need the word `Buttons` here

Reply:
I use this component only for demonstrating modal window, it has only button